### ISSUE TITLE
Implement support for directly accessing hardware registers

### ISSUE THREAD

**Issue by danicampora** [danicampora/micropython#18]
Implement something similar to `modstm.c`

**Comment by dpgeorge** [comment]
See https://github.com/micropython/micropython/issues/1085 which aims to make the stm.mem* accessors available to all ports.  On top of this you would then need to provide constants for the registers.

**Comment by danicampora** [comment]
Nice, thank you, I'll give it a look!

**Comment by danicampora** [comment]
I think this feature is another of the luxuries that the CC3200 can't afford.

**Comment by dpgeorge** [comment]
Yeah, it requires a lot of memory for the qstrs.  You can instead just have the mem8/mem16/mem32 functions, and then users can get addresses by importing some module with a list of constants.  But I'd say this is low priority for the moment.
